waku/v2/protocol/store/waku_store.go

```diff
@@ -632,3 +617,15 @@ func (store *WakuStore) Resume(ctx context.Context, pubsubTopic string, peerList
 func (w *WakuStore) Stop() {
 	w.h.RemoveStreamHandler(StoreID_v20beta3)
 }
+
+func recordMessageMetrics(ctx context.Context, tagType string, len int) {
+	if err := stats.RecordWithTags(ctx, []tag.Mutator{tag.Insert(metrics.KeyType, tagType)}, metrics.StoreMessages.M(int64(len))); err != nil {
+		log.Error("failed to record with tags", err)
+	}
+}
+
+func recordErrorMetric(ctx context.Context, tagType string) {
+	if err := stats.RecordWithTags(ctx, []tag.Mutator{tag.Insert(metrics.ErrorType, tagType)}, metrics.StoreErrors.M(1)); err != nil {
+		log.Error("failed to record with tags", err)
+	}
+}
```

Comment:
suggestion: move the recordSomething to metrics package and centralized the fact of recording?